config/vufind/contentsecuritypolicy.ini

```diff
@@ -0,0 +1,30 @@
+; Settings for Content Security Policy header, you can learn more here:
+; https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+[CSP]
+; It is possible to use CSP only in report mode. See report_uri setting below.
+report_only = false
+
+; The nonce (number used once) - unique number for each request, it is strongly
+; recommend to keep this setting on
+use_nonce = true
+
+; Directives, you can find available directives, on this page:
+; https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+; For evaluating of CSP you can use this tool provided by Google:
+; https://csp-evaluator.withgoogle.com/
+[Directives]
+default-src[] = "'none'"
+script-src[] = "'strict-dynamic'"
+script-src[] = "'unsafe-inline'"
+script-src[] = "http:"
+script-src[] = "https:"
+connect-src[] = "'self'"
+style-src[] = "'self'"
+img-src[] = "'self'"
+font-src[] = "'self'"
+base-uri[] = "'self'"
+; Set URI which should browser use to report CSP violation, you should provide url
+; when you enable report_only
+report-uri[] = 'https://example.report-uri.com'
```

Comment:
Should these be commented out by default so we don't try to send things to invalid URLs by default? (I'm not sure I fully understand their purpose, so please correct me if it makes no difference).

Reply:
You are right. When there is report-uri directive, the browser tries to send JSON-formatted report of any content security policy violence to defined uri. So i commented it out.

It could be possible to create a handler for such reporting in VuFind itself, but it is out of the scope of this PR. And may not be worth to spend time on it, as there are some at least two main possibilities how to get these report - commercial service report-uri.com and seek-oss csp-server as opensource alternative, written in Node.js, with Elasticsearch as backend - so relatively heavy application...

Reply:
I agree that building this functionality into VuFind is probably overkill, at least for the moment. Of course we can always consider it if there's a strong need in the future, but we might as well start by getting the basic functionality in place.

Perhaps it's worthwhile to mention these reporting tool options in the comments in the .ini file to help people understand how to use the setting.

Reply:
I've added notes about the available software/service in the wiki.